src/main/java/org/sagebionetworks/bridge/AuthUtils.java

```diff
@@ -177,7 +177,7 @@
     public static final AuthEvaluator CAN_READ_STUDIES = new AuthEvaluator()
             .isEnrolledInStudy().or()
             .canAccessStudy().hasAnyRole(STUDY_DESIGNER, STUDY_COORDINATOR, ORG_ADMIN).or()
-            .hasAnyRole(DEVELOPER, RESEARCHER, ADMIN);
+            .hasAnyRole(DEVELOPER, RESEARCHER, WORKER, ADMIN);
```

Comment:
This could use a worker access test in AuthUtilsTest.

Reply:
Added